modules/osd-monitoring-default-monitoring-targets.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-understanding-the-monitoring-stack.adoc
+
+[id="default-monitoring-targets_{context}"]
+= Default monitoring targets
+
+In addition to the components of the stack itself, the default monitoring stack monitors:
+
+* CoreDNS
+* Elasticsearch (if Logging is installed)
+* etcd
+* Fluentd (if Logging is installed)
+* HAProxy
+* Image registry
+* Kubelets
+* Kubernetes apiserver
+* Kubernetes controller manager
+* Kubernetes scheduler
+* Metering (if Metering is installed)
+* OpenShift apiserver
+* OpenShift controller manager
+* Operator Lifecycle Manager (OLM)
+
+[NOTE]
+====
+Each {product-title} component is responsible for its monitoring configuration. For problems with the monitoring of an {product-title} component, open a bug in Bugzilla against that component, not against the general monitoring component.
```

Comment:
I think "an {product-title}" renders as "an Red Hat OpenShift Service on AWS" in this case. If this module is used anywhere (I don't see it included in an assembly anymore), then you could change this to "a {product-title}" .

Reply:
I module this topic from the assembly. Although information, it's not applicable to managed services.